src/Common/Exceptions/ServiceException.php

```diff
@@ -86,7 +86,12 @@ protected static function parseErrorMessage(ResponseInterface $response)
         $serializer = new XMLSerializer();
         $errorMessage = '';
         try {
+            $internalErrors = libxml_use_internal_errors(true);
             $parsedArray = $serializer->unserialize($response->getBody());
+            if (libxml_get_errors()) {
+                throw new \Exception('Cannot parse XML');
```

Comment:
Alternatively, you can construct the error message got from `libxml_get_errors()` and pass it out so user could get a clearer message.